design-documents/batch-query-services.md

```diff
@@ -0,0 +1,77 @@
+#Batch query services
+
+## Purpose
+
+The main reason for such change is performance.
+The system triggers multiple queries during an operation (page load, service call). 
+Instead of making several requests for data retrieval,
+it would be more efficient to assemble all requests and send them as the single call to query service.
```

Comment:
I think it's worth to say that we are assembling multiple requests of the same type

Reply:
fixed